profile/tradfri.yml

```diff
@@ -0,0 +1,120 @@
+# OpenRemote v3
```

Comment:
Think we discussed putting additional settings commented out in the `deploy.yml` profile with comments explaining what they are for so we can get rid of this profile file